### ISSUE TITLE
Feature request: MSBuild, Fsharp, .NET Core

### ISSUE THREAD

**Issue by ijean** [chromebrew/chromebrew#2564]
Hello,
thanks for adding Mono to the Chromebrew packages!

Please add, if possible, MSBuild and F#. I've tried to build myself F# using the provided Mono package, but it seems to have a hard dependency on MSBuild (which I've also tried to build without success).

Your sister project on macOS, Homebrew, builds F# into Mono, see [https://github.com/Homebrew/homebrew-core/blob/master/Formula/mono.rb](https://github.com/Homebrew/homebrew-core/blob/master/Formula/mono.rb).

Thanks for all your hard work.
Jean

**Comment by JL2210** [comment]
This is more based on Linuxbrew than Homebrew. Don't try to compare us to them, though. These projects have gone their separate ways. Try building GCC with only the fortran target.

**Comment by ijean** [comment]
@JL2210 Thanks, noted about Linuxbrew and Chromebrew. 

What has GCC and Fortran to do with with building MSBuild and F# ?

**Comment by codekrafter** [comment]
Would dotnet core work?

**Comment by JL2210** [comment]
No. If you're talking about the proprietary implementation by Microsoft, then we can't package that.

**Comment by codekrafter** [comment]
I am not talking about packaging it, but they could download binaries from the Microsoft site. It is also not propriety, it is open source. [https://github.com/dotnet/core](https://github.com/dotnet/core)

**Comment by cstrouse** [comment]
.NET Core is MIT licensed. Working on a package for this now.
